src/status.ts

```diff
@@ -1,58 +1,60 @@
-export const NOT_ACCEPTABLE: HttpStatus = { code: 406, phrase: "Not Acceptable" };
-export const PROXY_AUTHENTICATION_REQUIRED: HttpStatus = { code: 407, phrase: "Proxy Authentication Required" };
-export const REQUEST_TIMEOUT: HttpStatus = { code: 408, phrase: "Request Timeout" };
-export const CONFLICT: HttpStatus = { code: 409, phrase: "Conflict" };
-export const GONE: HttpStatus = { code: 410, phrase: "Gone" };
-export const LENGTH_REQUIRED: HttpStatus = { code: 411, phrase: "Length Required" };
-export const PRECONDITION_FAILED: HttpStatus = { code: 412, phrase: "Precondition Failed" };
-export const REQUEST_TOO_LONG: HttpStatus = { code: 413, phrase: "Request Entity Too Large" };
-export const REQUEST_URI_TOO_LONG: HttpStatus = { code: 414, phrase: "Request-URI Too Long" };
-export const UNSUPPORTED_MEDIA_TYPE: HttpStatus = { code: 415, phrase: "Unsupported Media Type" };
-export const REQUESTED_RANGE_NOT_SATISFIABLE: HttpStatus = { code: 416, phrase: "Requested Range Not Satisfiable" };
-export const EXPECTATION_FAILED: HttpStatus = { code: 417, phrase: "Expectation Failed" };
-export const IM_A_TEAPOT: HttpStatus = { code: 418, phrase: "I'm a teapot" };
-export const INSUFFICIENT_SPACE_ON_RESOURCE: HttpStatus = { code: 419, phrase: "Insufficient Space on Resource" };
-export const METHOD_FAILURE: HttpStatus = { code: 420, phrase: "Method Failure" };
-export const MISDIRECTED_REQUEST: HttpStatus = { code: 421, phrase: "Misdirected Request" };
-export const UNPROCESSABLE_ENTITY: HttpStatus = { code: 422, phrase: "Unprocessable Entity" };
-export const LOCKED: HttpStatus = { code: 423, phrase: "Locked" };
-export const FAILED_DEPENDENCY: HttpStatus = { code: 424, phrase: "Failed Dependency" };
-export const PRECONDITION_REQUIRED: HttpStatus = { code: 428, phrase: "Precondition Required" };
-export const TOO_MANY_REQUESTS: HttpStatus = { code: 429, phrase: "Too Many Requests" };
-export const REQUEST_HEADER_FIELDS_TOO_LARGE: HttpStatus = { code: 431, phrase: "Request Header Fields Too Large" };
-export const UNAVAILABLE_FOR_LEGAL_REASONS: HttpStatus = { code: 451, phrase: "Unavailable For Legal Reasons" };
-export const INTERNAL_SERVER_ERROR: HttpStatus = { code: 500, phrase: "Internal Server Error" };
-export const NOT_IMPLEMENTED: HttpStatus = { code: 501, phrase: "Not Implemented" };
-export const BAD_GATEWAY: HttpStatus = { code: 502, phrase: "Bad Gateway" };
-export const SERVICE_UNAVAILABLE: HttpStatus = { code: 503, phrase: "Service Unavailable" };
-export const GATEWAY_TIMEOUT: HttpStatus = { code: 504, phrase: "Gateway Timeout" };
-export const HTTP_VERSION_NOT_SUPPORTED: HttpStatus = { code: 505, phrase: "HTTP Version Not Supported" };
-export const INSUFFICIENT_STORAGE: HttpStatus = { code: 507, phrase: "Insufficient Storage" };
-export const NETWORK_AUTHENTICATION_REQUIRED: HttpStatus = { code: 511, phrase: "Network Authentication Required" };
+export abstract class HttpStatusCode {
```

Comment:
Hey, what's the benefit of having a class vs a simple module?

Reply:
the class format could be better for encapsulation and organization.